pennylane/templates/constructors/broadcast_double.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2018-2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the ``broadcast`` template constructor.
+"""
+# pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from collections import Iterable
+
+from pennylane.templates.decorator import template
+from pennylane.templates.utils import _check_wires, _check_type, _get_shape
+
+
+@template
+def broadcast_double(block, wires, parameters=None, even=True, kwargs={}):
+    r"""Applies a two-qubit unitary to pairs of wires.
+
+    If ``even`` is ``True``, start with the wire pair ``[0, 1]``, else with the pair ``[1, 2]``.
```

Comment:
This part gives the impression that these lists that are used as wires when applying the blocks (whereas if I understand it correctly, these are *indices*)